stack-lts-11.yaml

```diff
@@ -2,3 +2,5 @@ resolver: lts-11.22
 packages:
 - unliftio
 - unliftio-core
+extra-deps:
+- gauge-0.2.4@sha256:b8b19a8c13ab79097726d0edf91297ccd3eede053d1de47e3ac67f1252cc33c2
```

Comment:
Since you've disabled `--bench` for Travis, do we still need these `extra-deps` for gauge?